prow/plugins/releasenote/releasenote.go

```diff
@@ -235,11 +231,15 @@ func handlePR(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent) e
 		log.Error(err)
 	}
 
-	return clearStaleComments(gc, log, pr, prLabels, comments)
+	return clearStaleComments(gc, log, pr, comments)
 }
 
-func clearStaleComments(gc githubClient, log *logrus.Entry, pr *github.PullRequestEvent, prLabels []github.Label, comments []github.IssueComment) error {
-	// Clean up old comments.
+// Clean up old comments.
```

Comment:
go doc comment should start with `clearStaleComments`
`// clearStaleComments deletes old comments that are no longer applicable.`

Reply:
Thanks for pointing out!